src/components/application_manager/src/commands/mobile/show_constant_tbt_request.cc

```diff
@@ -181,6 +181,7 @@ void ShowConstantTBTRequest::on_event(const event_engine::Event& event) {
   switch (event.id()) {
     case hmi_apis::FunctionID::Navigation_ShowConstantTBT: {
       LOG4CXX_INFO(logger_, "Received Navigation_ShowConstantTBT event");
+      const char* return_info = NULL;
```

Comment:
@anosach-luxoft, actually I like the existing approach. Your variant with additional string allocation, ternary operator looks worse to me. The pointer to internals doesn't looks to scary to me since we know for sure that the massage will not change. I don't like most the assignment to the const pointer. But I can leave with this fact.